sitegen/src/main/java/io/helidon/sitegen/freemarker/TemplateLoader.java

```diff
@@ -57,7 +57,11 @@ public TemplateLoader(){
 
     @Override
     protected URL getURL(String name) {
-        Path tpl = templatesDir.resolve(name + TEMPLATE_FILE_EXT);
+        String tplName = name;
+        if (!tplName.endsWith(TEMPLATE_FILE_EXT)) {
```

Comment:
Is there ever a case in which the template has some _other_ extension and we would _not_ want to add `.ftl` automatically? Put another way, should this logic add `.ftl` only if there is no extension already present?

Reply:
Template files could be named `foo.txt.ftl` or `foo.vue.ftl`, in this case we'd want both `foo.txt` and `foo.vue` to resolve to `foo.txt.ftl` or `foo.vue.ftl`. Also I can't think of a case where we'd have freemarker without the .ftl extension.